src/library/scala/concurrent/duration/Duration.scala

```diff
@@ -494,41 +494,41 @@ sealed abstract class Duration extends Serializable with Ordered[Duration] {
    *
    * $ovf
    */
-  def div(divisor: Double)    = this / divisor
+  def div(divisor: Double): Duration = this / divisor
   /**
    * Return the quotient of this and that duration as floating-point number. The semantics are
    * determined by Double as if calculating the quotient of the nanosecond lengths of both factors.
    */
-  def div(other: Duration)   = this / other
-  def gt(other: Duration)    = this > other
-  def gteq(other: Duration)  = this >= other
-  def lt(other: Duration)    = this < other
-  def lteq(other: Duration)  = this <= other
+  def div(other: Duration): Double = this / other
+  def gt(other: Duration): Boolean = this > other
+  def gteq(other: Duration): Boolean = this >= other
+  def lt(other: Duration): Boolean = this < other
+  def lteq(other: Duration): Boolean = this <= other
```

Comment:
Text used to be aligned on `=`. The types are helpful here.

Reply:
I  fixed up the alignment